### PR TITLE
Added Stargate Protocol

### PR DESCRIPTION
**Pull request by cryptonoyaiba**
- Added Stargate as CrossChainServiceProvider.
- Old branch - merged master and fixed conflicts.

### CODE REVIEW

**Review by ch4r10t33r — APPROVED** [review]

**Review by 0xSulpiride — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "etherspot",
-  "version": "1.35.2",
+  "version": "1.37.0",
```
Comment (vignesha22) [inline package.json:3]: shouldn't it be 1.36 as the next version available?

Comment (vignesha22) [reply package.json:3]: Okay got it so pls merge it after hop integration

**Review by vignesha22 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (lbw33) [reply package.json:3]: If Hop is merged first then (which is 1.36.0) then this should follow on from it

**Comment by lbw33** [comment]
TODO: bump package version

**Review by ch4r10t33r — APPROVED** [review]

**Pull request closed without merging**